### DISCUSSION TITLE
will there be an app for AppleTV?

### DISCUSSION THREAD

**Discussion by fahadshery** [photoprism/photoprism#1883] [Ideas]
Hi,

I use `plex` for photos. I love this app. Would there be an `AppleTV` app? 

Thanks

**Comment by graciousgrey** [comment]
We already have an issue about it: https://github.com/photoprism/photoprism/issues/1513

So it might be added at one point, but currently there are many tasks with higher priority on our roadmap.

**Comment by fahadshery** [comment]
Is there any potential ETA? Will it be this year you reckon?

**Reply by graciousgrey** [reply]
We have a zero bug policy and do our best to help users when they need support or have other questions. Updating and improving the documentation  takes up a large part of our time too.

This comes at a price, as we can't give exact deadlines for new features. Also keep in mind that epics like multi-user support can't be delivered in a single update. Working on big features is a continuous process that can take months to years until all use cases are covered.

Having said that, funding really has the highest impact. So users can do their part and become a sponsor to get their favorite features as soon as possible.
